src/markup/codemirror/create.ts

```diff
@@ -112,7 +114,7 @@ export function createCodemirror(params: CreateCodemirrorParams) {
             ...defaultKeymap,
             ...historyKeymap,
         ]),
-        autocompletion(),
```

Comment:
Let us revert to the original code; it would seem that there is no longer a need for overriding options.